transformers/shared/constants/table.go

```diff
@@ -65,3 +66,62 @@ const (
 	VowFlogTable            = "vow_flog"
 	YankTable               = "yank"
 )
+
+// storage tables
+const (
+	CdpManagerVat      = "cdp_manager_vat"
```

Comment:
Would be a fan of adding `Table` to the end of these names so it's clear what purpose they serve when imported in other packages, though not sure whether that could cause collisions with the event tables?